apps/vaults/hooks/useSolverCowswap.ts

```diff
@@ -276,22 +302,35 @@ export function useSolverCowswap(): TSolverContext {
 			return (toNormalizedBN(0));
 		}
 		const quoteResult = await getQuote(_request, true);
-		return toNormalizedBN(formatBN(quoteResult?.quote?.buyAmount), _request.outputToken.decimals);
+		return (
+			toNormalizedBN(
+				toBigInt(quoteResult?.quote?.buyAmount.toString()),
```

Comment:
Should `toBigInt` accept `BigNumberish`? So we don't need to convert to `string`?